pkg/kademlia/kademlia.go

```diff
@@ -141,7 +142,7 @@ func (k *Kademlia) DumpNodes(ctx context.Context) (_ []*pb.Node, err error) {
 
 // Bootstrap contacts one of a set of pre defined trusted nodes on the network and
 // begins populating the local Kademlia node
-func (k *Kademlia) Bootstrap(ctx context.Context) (err error) {
+func (k *Kademlia) Bootstrap(ctx context.Context, transport transport.Client) (err error) {
```

Comment:
This probably shouldn't need a separate argument for transport client?

Reply:
With the dialer moved to another package, the transport field on it is not reachable from inside `Bootstrap`
https://github.com/storj/storj/pull/2466/files#diff-f6eb0f0cefab0c21ae61c0e1022a854eL186